core-web/libs/dotcms-models/src/lib/dot-feature-flags.constants.ts

```diff
@@ -0,0 +1,10 @@
+import { FeaturedFlags } from './shared-models';
+
+export const ENABLED_FEATURE_FLAGS = [
+    FeaturedFlags.LOAD_FRONTEND_EXPERIMENTS,
+    FeaturedFlags.DOTFAVORITEPAGE_FEATURE_ENABLE,
+    FeaturedFlags.FEATURE_FLAG_TEMPLATE_BUILDER,
+    FeaturedFlags.FEATURE_FLAG_SEO_IMPROVEMENTS,
+    FeaturedFlags.FEATURE_FLAG_SEO_PAGE_TOOLS,
+    FeaturedFlags.FEATURE_FLAG_EDIT_URL_CONTENT_MAP
+];
```

Comment:
With this we will have to modify the enum and this array to have all the pieces updated. Can we find a way to just transform the enum into an array of keys dynamically?

Reply:
I tested this and we can use `Object.values()` to get the values, but we need to remove the const keyword from the enum, that way the linter will not throw an error and `Object.values()` will work as expected.

<img width="532" alt="Screenshot 2023-09-27 at 11 16 42 AM" src="https://github.com/dotCMS/core/assets/63567962/51bbe734-1b9f-4b22-83ac-d58730cd6c9d">

The idea is to have a Single Source of Truth.

Reply:
Probably some Typescript magic.

Reply:
At the end, Enums is TypeScript are objects in JavaScript

Reply:
I don't think it should be a problem